### PR TITLE
Compiling with ghc 8.8

### PR DESCRIPTION
**Pull request by rezafarzin**

### CODE REVIEW

**Comment by alunduil** [comment]
This looks like a duplicate of #31.  I'll use that one if you don't mind.

**Comment by alunduil** [comment]
On second thought, they should merge just fine and this will update the travis configuration so I'll keep both.

File: network-uri-json.cabal [hunk 1]
```diff
@@ -75,7 +75,7 @@ test-suite network-uri-json-tests
       aeson             >= 0.8 && < 1.5
     , base              >= 4.6 && < 4.14
     , hspec             >= 2.4 && < 2.8
-    , network-arbitrary >= 0.3 && < 0.5
+    , network-arbitrary >= 0.3 && <= 0.6
```
Comment (alunduil) [inline network-uri-json.cabal]: For consistency use < 0.7.  That way we include any minor and patch level (non-API breaking) changes automatically.

File: network-uri-json.cabal [hunk 2]
```diff
@@ -75,7 +75,7 @@ test-suite network-uri-json-tests
       aeson             >= 0.8 && < 1.5
     , base              >= 4.6 && < 4.14
     , hspec             >= 2.4 && < 2.8
-    , network-arbitrary >= 0.5 && < 0.7
+    , network-arbitrary >= 0.4 && < 0.7
```
Comment (alunduil) [inline network-uri-json.cabal:78]: Why are we lowering the version on the lower bound?

**Comment by alunduil** [comment]
This looks like a duplicate of #31 at this point.  If it ends up as a no op after that lands I'll go ahead and close this one out.

**Comment by rezafarzin** [comment]
Duplicate #31

**Pull request closed without merging**